### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
##Feature objectives
Learn how to create a responsive design with Media Queries
Understand the importance of UX.
Keep learning basics of HTML & CSS
Understand how to parse a Figma design to create a UI.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -430,12 +434,123 @@ textarea {
 
 /* Media queries */
 
-@media only screen and (min-width: 800px) {
-  .card-container {
-    grid-template-columns: repeat(3, 1fr);
+@media all and (min-width: 850px) {
```
Comment (arslanbisharat) [inline style.css:437]: - [ ] The value for min-width property should be in range of 992px - 1100px.

File: index.html [hunk 2]
```diff
@@ -85,30 +99,33 @@ <h2 class="fw-normal">I am Hamza</h2>
...
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="languages">
+              <li><p>css</p></li>
+              <li><p>html</p></li>
+              <li><p>bootsrap</p></li>
+              <li><p>Ruby</p></li>
+            </ul>
```
Comment (arslanbisharat) [inline index.html:124]: - [ ] This is not matching with the origianl design. Please work on it. 

**Clone** 

Capture

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested , 

Hi @Hamzaoutdoors, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] Please DS_Store to .`gitignore`.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by tahiry-dev — APPROVED** [review]
Hello Hamzaoutdoors 👋, 

Congratulations! 🎉 , your project is complete!  You have done an amazing job, you have implemented all the requirements 💪💪.

[optional]: One little suggestion though. Next time when you write your PR description, please summarize the steps you took to accomplish the project (e.g: created a basic HTML page,  added flexbox to align elements,...)   instead of enumerating the objective of the project

You can now merge it and move to the next step!, 
 👉keep up the good work

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**